xExtension-ImageProxy/i18n/de/ext.php

```diff
@@ -5,7 +5,7 @@
 		'proxy_url' => 'Proxy-URL',
 		'scheme_http' => 'HTTP-Proxy',
 		'scheme_https' => 'HTTPS-Proxy',
-		'scheme_default' => 'Proxy unspezifiziert',
+		'scheme_default' => 'Proxy protokoll-relative URL',
```

Comment:
Also for reference:

https://www.seo-suedwest.de/3228-google-protokoll-relative-urls-fuer-canonical-links-ok-aber-nicht-emnpfohlen.html
https://de.wikipedia.org/wiki/Hilfe:Parserfunktionen/Seite_und_URL